### PR TITLE
[Coverity] Fix Coverity issue

### PR DESCRIPTION
**Pull request by DonghakPark**
## In this PR

Fix Coverity issue & Fix some typo

- Auto_causes_copy issue
- leaked_storage issue
- returned_null issue

""""""""""""""""""""""""""""""""""""""""""""""
**Self evaluation:**
_Build test: [x]Passed [ ]Failed [ ]Skipped
Run test: [x]Passed [ ]Failed [ ]Skipped_
""""""""""""""""""""""""""""""""""""""""""""""

Signed-off-by: Donghak PARK <donghak.park@samsung.com>

### CODE REVIEW

**Review by djeong20 — APPROVED** [review]
LGTM

File: Applications/SimpleShot/task_runner.cpp [hunk 1]
```diff
@@ -104,14 +104,19 @@ const std::string getcwd_() {
   const size_t bufsize = 4096;
   char buffer[bufsize];
 
-  return getcwd(buffer, bufsize);
+  if (getcwd(buffer, bufsize) == nullptr) {
```
Comment (myungjoo) [inline Applications/SimpleShot/task_runner.cpp:107]:
```
if ((retvalue = getcwd(...)) == nullptr) {
  ml_loge("...");
  return "";
} else {
  return retvalue;
}
```
?

Comment (DonghakPark) [reply Applications/SimpleShot/task_runner.cpp:107]: this code check getcwd work well

**Review by songgot — APPROVED** [review]
LGTM

**Review by skykongkong8 — APPROVED** [review]
LGTM!

**Review by jijoongmoon — APPROVED** [review]

**Pull request merged**